clients/csharp-dotnet-core/voice-assistant-test/README.md

```diff
@@ -277,6 +281,23 @@ If the keyword has been recognized successfully, the identified keyword will be
 
 Note that due to a bug in the way Speech SDK consumes audio from an input stream, keyword activation is limited to the first turn of the dialog. Therefore [Keyword](#keyword) can only be set to true when [TurnID](#turnid) is 0.
 
+### Using RealTimeAudio

```

Comment:
Call this section "Measuring User Perceived Latency (UPL)"

Reply:
Done